build-tools/rpm/macros.yast

```diff
@@ -79,7 +79,7 @@ fi \
     %if %{with yast_run_ci_tests} \
       if [ -f "configure.in.in" -o -f "configure.ac.in" ]; then \
           # TODO: fix the check:ci task to also work with autotools based modules \
-          %{__make} check \\\
+          %{__make} check %{?jobs:-j%jobs}%{?!jobs:%{?_smp_mflags:%_smp_mflags}} \\\
```

Comment:
I can hardly understand that, but it's the same as for the normal `make`. So I'm fine with that. :wink: